lib/performance_platform/gateway/sequel_ip_locations.rb

```diff
@@ -0,0 +1,15 @@
+class PerformancePlatform::Gateway::SequelIPLocations
+  def save(ip_locations)
```

Comment:
This save+ truncate isn't in a transaction, which means if it falls over halfway through it won't recover back to the untruncated version. Can add test + behaviour, but was worried about time.

It's also looping through the ip/locations and making a separate insert for each one, which is inefficient, but probably fine at the scale we're expecting.

Reply:
I've had a quick look in the Sequel documentation but I can't see anything about saving an array of hashes. Surely its possible? Its already in the right format.

Reply:
Better be explicit IMO.